### PR TITLE
Fixed CORE management port issue (micro-services health info)

### PR DESCRIPTION
**Pull request by shariq1501**

### CODE REVIEW

File: src/frontend/src/app/settings/settings.component.ts [hunk 1]
```diff
@@ -14,17 +14,31 @@ export class SettingsComponent implements OnInit {
...
   host = this.endpoint[1].substr(2);
   port = this.endpoint[2].substring(0, this.endpoint[2].indexOf('/'));
+  management_port = 0;
   constructor(private router: Router) { }
 
-  ngOnInit() { }
+  ngOnInit() {
+    if (environment.MANAGEMENT_URL !== '') {
+      const url_items = environment.MANAGEMENT_URL.split(':');
+      console.log(url_items);
```
Comment (praveen-garg) [inline src/frontend/src/app/settings/settings.component.ts]: remove this log

File: src/frontend/src/environments/environment.prod.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export const environment = {
   production: true,
-  BASE_URL: localStorage.getItem('API_END_POINT') == null ? 'http://0.0.0.0:8082/foglamp/' : localStorage.getItem('API_END_POINT')
+  BASE_URL: localStorage.getItem('SERVICE_URL') == null ? 'http://0.0.0.0:8082/foglamp/' : localStorage.getItem('SERVICE_URL'),
```
Comment (praveen-garg) [inline src/frontend/src/environments/environment.prod.ts]: 'http://0.0.0.0:8082/foglamp/ make 8082 to 8081 rest is valid for prod

File: src/frontend/src/app/services-health/services-health.component.ts [hunk 3]
```diff
@@ -11,50 +11,20 @@ import { environment } from '../../environments/environment';
...
   public getServiceData() {
-    this.servicesHealthService.getServicesData()
+    this.servicesHealthService.getAllServices()
       .subscribe(
       (data) => {
-        console.log('Service Status', data.services);
         this.service_data = data.services;
       },
       (error) => {
         console.log('error: ', error);
```
Comment (praveen-garg) [inline src/frontend/src/app/services-health/services-health.component.ts]: if error then  (if could not connect; show warning message "Could not connect to Core Managment API, Make sure to set correct [core management port](/settings)" with link to navigate to settings page to fill up core port)

File: src/frontend/src/app/settings/settings.component.html [hunk 4]
```diff
@@ -13,14 +13,18 @@
       <label class='label'>Host</label>
       <input id='host' class='input' type='text' placeholder='0.0.0.0' [ngModel]='host'>
     </div>
-     <div class='column'>
+    <div class='column'>
       <label class="label">Port</label>
-      <input id='port' class='input' type='numbe' maxlength="4" placeholder='8082' [ngModel]='port'>
+      <input id='port' class='input' type='number' maxlength="4" placeholder='8081' [ngModel]='port'>
```
Comment (praveen-garg) [inline src/frontend/src/app/settings/settings.component.html]: if possible call it id='service_port'

**Review by praveen-garg — CHANGES_REQUESTED** [review]
minor changes! only log the stuff which matters

**Comment by praveen-garg** [comment]
just below or next to "Services Status" add a light color tag with `Last updated at: Jan 26, 2017 15:23:12`

**Comment by shariq1501** [comment]
@praveen-garg Fixed per feedback. Please review now.

File: src/frontend/src/app/directives/alert.component.html [hunk 5]
```diff
@@ -1,3 +1,3 @@
-<div id = "alert" *ngIf="message" [ngClass]="{ 'notification': message, 'is-success': message.type === 'success', 'is-danger': message.type === 'error' }">{{message.text}}
+<div [innerHTML]="message.text" id = "alert" *ngIf="message" [ngClass]="{ 'notification': message, 'is-success': message.type === 'success', 'is-danger': message.type === 'error','is-warning': message.type === 'warning' }">{{message.text}}
```
Comment (praveen-garg) [inline src/frontend/src/app/directives/alert.component.html]: do we need {{message.text}} as innerText now? as we have innerHTML

File: src/frontend/src/app/utils.ts [hunk 6]
```diff
@@ -80,4 +80,8 @@ export default class Utils {
     console.log('Total sec', totalSec);
     return totalSec < 0 || totalSec >= 86400;
   }
+
+  public static getCureentDate() {
```
Comment (praveen-garg) [inline src/frontend/src/app/utils.ts]: getCurrentDate

**Review by praveen-garg — COMMENTED** [review]
1 question; 1 minor change and we are good to go

**Comment by praveen-garg** [comment]
clear the local storage and start new; I don't see any warning on services-health page

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**